src/components/map/mapControls/MapLayersControl.tsx

```diff
@@ -88,7 +89,14 @@ class MapLayersControl extends React.Component<
                         toggleSelectedTransitType={this.toggleTransitType}
                         selectedTransitTypes={NetworkStore.selectedTransitTypes}
                     />
-                    <NetworkDateControl selectDate={this.selectDate} />
+                    <NetworkDateControl
```

Comment:
Isnt NetworkDateControl a bit overkill? Maybe just use DatePicker straightly?

Reply:
Why do we btw need DatePicker anyway? Can't we just use InputContainer with type='date'? And then DatePicker.tsx could be removed also